CMakeLists.txt

```diff
@@ -7,24 +7,69 @@ include(ExternalProject)
 
 project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
 
+# Set build path as a folder named as the platform (linux, windows, darwin...) plus the processor type
+set(EXECUTABLE_OUTPUT_PATH "${PROJECT_BINARY_DIR}/${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}")
+set(LIBRARY_OUTPUT_PATH "${PROJECT_BINARY_DIR}/${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}")
```

Comment:
EXECUTABLE_OUTPUT_PATH and LIBRARY_OUTPUT_PATH seems to be unused.

Reply:
yes, that's a mistake, we should use the `CMAKE_` prefixed version to initialize them. Also, they have been replaced by CMAKE_RUNTIME_OUTPUT_DIRECTORY and CMAKE_LIBRARY_OUTPUT_DIRECTORY. Fixing it